docs/example-react-hooks-useContext.md

```diff
@@ -0,0 +1,95 @@
+This is a simple example to test a component using the useContext hook. 
+
+
+The context object
+
+```
+import React, { createContext } from 'react'
+
+const Context = createContext()
+
+export default Context
+```
+
+the root parent component
+
+```
+import React, { useState } from 'react'
+import Context from './context'
+
+
+const App = () => {
+  const [state, setState] = useState("Some Text")
+
+  const changeText = () => {
+    setState("Some Other Text")
+  }
+
+   <h1> Basic Hook useContext</h1>
```

Comment:
isn't a `return` statement missing here? I'd run all code in docs, to make sure it is working properly. We don't want to provide broken demos to our readers! 😉

Reply:
Wow your right cant believe I missed something so obvious